test/models/TaxRateUtil.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import org.scalacheck.Gen
+
+object TaxRateUtil {
+  val janToMarInt: Gen[Int] = Gen.choose(1, 3)
+  val aprToDecInt: Gen[Int] = Gen.choose(4, 12)
+
+  val lowerBandCostPerLitre: BigDecimal = BigDecimal("0.18")
+  val higherBandCostPerLitre: BigDecimal = BigDecimal("0.24")
+
+  val lowerBandCostPerLitreMap: Map[Int, BigDecimal] = Map(2025 -> BigDecimal("0.194"))
+  val higherBandCostPerLitreMap: Map[Int, BigDecimal] = Map(2025 -> BigDecimal("0.259"))
```

Comment:
Also need to consider the view unit tests and updates required. See Zahra's PR for work on this https://github.com/hmrc/soft-drinks-industry-levy-variations-frontend/pull/305